examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -88,18 +94,19 @@
 print(f"Using plotting engine {engine}.")
 
 figure = plot_surf_stat_map(
-    stat_map=img,
+    stat_map=surface_image,
     surf_mesh=fsaverage_meshes["inflated"],
-    hemi="right",
-    title="Surface with plotly",
+    hemi=hemi,
+    title=f"Surface with {engine}",
     colorbar=True,
     threshold=1.0,
-    bg_map=curv_right_sign,
+    bg_map=curv_sign,
     bg_on_data=True,
     engine=engine,  # Specify the plotting engine here
 )
 
-# Uncomment to display the figure
+# Uncomment the line below
+# to view the figure in browser.
 # figure.show()
```

Comment:
In the previous stable doc, the plot was rendered...

Reply:
- yeah we have this issue for a while on the doc build https://github.com/nilearn/nilearn/issues/4379